kedro-training/docs/01_prerequisites.md

```diff
@@ -1,25 +1,81 @@
+  - Common data structures: lists, dictionaries, tuples
+- Intermediate Python
+  - [Installing Python packages using `pip`](https://pip.pypa.io/en/stable/quickstart/)
+  - [Dependency management with `requirements.txt`](https://pip.pypa.io/en/latest/user_guide/#requirements-files)
+  - [Python modules](https://docs.python.org/3/tutorial/modules.html) (e.g how to use `__init__.py` and relative and absolute imports)
+  - [Some familiarity with Python data science libraries](https://towardsdatascience.com/top-10-python-libraries-for-data-science-cd82294ec266), especially `Pandas`, `scikit-learn`, [Jupyter notebook/labs](https://www.dataquest.io/blog/jupyter-notebook-tutorial/), [iPython](https://www.codecademy.com/articles/how-to-use-ipython))
+  - [Using a virtual environment](https://docs.python.org/3/tutorial/venv.html) (we recommend using `conda`, but you can also use `venv` or `pipenv`)
+- [Basic YAML syntax](https://yaml.org/)
+- [Working with the command line](https://tutorial.djangogirls.org/en/intro_to_command_line/) (aka. cmd, CLI, prompt, console or terminal)
+  -  `cd` to navigate directories
+  -  `ls` to list files and directories
+  -  [Executing a command and Python program from a command line](https://realpython.com/run-python-scripts/#how-to-run-python-scripts-using-the-command-line)
+
+The following lists technologies that Kedro integrates with, but it won't cause any issues if you are not yet familiar with them.
+- [Version Control with Git](https://git-scm.com/doc)
+- Cloud storage ([S3](https://aws.amazon.com/s3/), [Azure Blob](https://azure.microsoft.com/en-gb/services/storage/blobs/), [GCS](https://cloud.google.com/storage))
+- [PySpark](https://spark.apache.org/docs/latest/api/python/index.html)
+- [Docker](https://docs.docker.com/) for managing deployment
+- [Airflow](https://airflow.apache.org/docs/stable/tutorial.html) for scheduling pipeline execution
+
+# Checklist
+Please use this checklist to make sure you have everything necessary to participate in the Kedro training.
+
+- [ ] You have [Python 3 (either 3.6, 3.7 or 3.8)](https://www.python.org/downloads/) installed in your laptop.
+- [ ] You have Anaconda or an [alternative](https://github.com/quantumblacklabs/kedro-examples/blob/master/kedro-training/docs/02_virtual-environment.md) virtual environment manager
+- [ ] You have [an editor](#text-editors-and-ides) installed for writing Python code
+- [ ] You have a Terminal installed
+
+Having completed the above checklist, make sure that you are able to execute the following commands in your Terminal
+- [ ]  `python --version` or `python3 --version` returns a correct Python version (either 3.6, 3.7 or 3.8).
+- [ ] `git clone https://github.com/quantumblacklabs/kedro-examples` downloads this training Github repository (or [manually download a zip file from the link](https://stackoverflow.com/questions/2751227/how-to-download-source-in-zip-format-from-github)).
```

Comment:
This makes an assumption that they've done the `git` work. Rather just tell tell them to download the repo, so they all have one way to do it.

```suggestion
- [ ] Download the [Kedro training repository](https://github.com/quantumblacklabs/kedro-examples) by following [these instructions](https://stackoverflow.com/questions/2751227/how-to-download-source-in-zip-format-from-github).
```